docs/sources/features/datasources/loki.md

```diff
@@ -36,6 +36,21 @@ Just add it as a data source and you are ready to query your log data in [Explor
 | _URL_           | The URL of the Loki instance, e.g., `http://localhost:3100`                                                                                   |
 | _Maximum lines_ | Upper limit for number of log lines returned by Loki (default is 1000). Decrease if your browser is sluggish when displaying logs in Explore. |
 
+### Derived fields
+
+Derived fields configuration allow you to add additional fields parsed from the log message and optionally add link that uses the value of the field. This can be used for example to link to trace in your tracing backend from your logs or user profile page if userId is present int the log line. They are then showed in the [log details](/features/explore/#labels-and-parsed-fields)
+{{< docs-imagebox img="/img/docs/v65/loki_derived_fields.png" class="docs-image--no-shadow" caption="Screenshot of the derived fields configuration" >}}
+Each derived field consists of:
+- Name: will be shown in the log details as label
+- Regex: regex pattern that will be run on the log message and should capture some part of it to be used as the value of the new field. Can only contain single capture group.
+- URL: a url template that will be used to construct a link next to the field value in log details. Use special `${__value.raw}` to insert the value of the field.
```

Comment:
```suggestion
- URL: A URL template used to construct a link next to the field value in log details. Use special `${__value.raw}` to insert the value of the field.
```

Reply:
Is the second sentence in this bullet instructing the reader to **use** the command to insert the value or saying that the template **uses** the command to insert the value?

Reply:
It is used for interpolating the real value into the URL string. Like: `http://trace-service/trace/${__value.raw}` will later replace `${__value.raw}` with the value of the field. So maybe something like:
> Use special `${__value.raw}` value in your template to interpolate the real field value into your URL template.

Reply:
Good idea!